portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -33,32 +36,45 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
         Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
 
         PreparedQuery results = DatastoreServiceFactory.getDatastoreService().prepare(query);
 
         List<Comment> comments = new ArrayList<>();
+
         for (Entity entity : results.asIterable()) {
             String name = (String) entity.getProperty("name");
             String commentText = (String) entity.getProperty("commentText");
             long timestamp = (long) entity.getProperty("timestamp");
+            double score = (double) entity.getProperty("score");
 
-            Comment currComment = new Comment(name, commentText, timestamp);
+            Comment currComment = new Comment(name, commentText, timestamp, score);
             comments.add(currComment);
         }
 
         Gson gson = new Gson();
         response.setContentType("application/json;");
         response.getWriter().println(gson.toJson(comments));
     }
+
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        String newComment = request.getParameter("new-comment");
         Entity commentEntity = new Entity("Comment");
         
+        Document doc =
+        Document.newBuilder().setContent(newComment).setType(Document.Type.PLAIN_TEXT).build();
+        LanguageServiceClient languageService = LanguageServiceClient.create();
+        Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
```

Comment:
Optional: You can remove sentiment variable and directly find score.

Reply:
I will make this change